aws-sdk-core/lib/aws-sdk-core/xml/parser.rb

```diff
@@ -23,9 +23,9 @@ def initialize(rules, options = {})
       # @param [String] xml An XML document string to parse.
       # @param [Structure] target (nil)
       # @return [Structure]
-      def parse(xml, target = nil)
+      def parse(xml, target = nil, &unhandled_callback)
```

Comment:
I understand what `&unhandled_callback` is for because of the full context of the change, but it could be hard to remember or not intuitive to someone reading. Can we add this to the docs above?